Rakefile

```diff
@@ -18,11 +16,11 @@ end
 
 task :proof do
   HTMLProofer.check_directory(
-    './_site', \
-    assume_extension: true, \
-    check_html: true, \
-    disable_external: true, \
-    cache: { timeframe: '2d', storage_dir: '/tmp/html-proofer' }
+    './_site',
+    assume_extension: true,
+    check_html: true,
+    disable_external: true,
+	parallel: { in_threads: 5 }
```

Comment:
The "parallel" line is indented by a tab instead of 4 spaces, so "rubocop" fails:

> Offenses:
> 
> Rakefile:23:1: C: Layout/Tab: Tab detected.
> 	parallel: { in_threads: 5 }
> ^
> Rakefile:23:2: C: Layout/AlignHash: Align the elements of a hash literal if they span more than one line.
> 	parallel: { in_threads: 5 }
>  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 
> 2 files inspected, 2 offenses detected
> RuboCop failed!
>